### ISSUE TITLE
Execution fails with 'An unexpected error has occurred'

### ISSUE THREAD

**Issue by d0tkom** [InfoSec812/npm-audit-ci-wrapper#6]
For some of my projects running ```npm-audit-ci-wrapper``` fails with 'An unexpected error has occurred'. I looked into the code, and was wondering what's the reason for throwing this error here if ```err``` is null? 
```exec('npm audit --json', (err, stdout, stderr) => {
  let exitCode = 0;
  if (err === null) {
    console.log('An unexpected error has occurred')
    console.log(stderr);
    exitCode = 255;
  } else {
```

as far as I know if everything goes correctly, ```err``` should be null no?

**Comment by InfoSec812** [comment]
Interesting... I checked the docs on the [exec](https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback) method and your reasoning makes sense, but for some reason it has been working for me all of this time. Could you propose a PR and we can have a look?

**Comment by TazSp** [comment]
`npm audit` returns success when there are no vulnerabilities as an exit code 0 which would make `err` be `null`. So (at least for me) `npm-audit-ci-wrapper` throws "An unexpected error has occurred" when there are no vulnerabilities. It seems logical `npm-audit-ci-wrapper` should exit with 0 in this case (which makes it useful in a CI pipeline).

**Comment by InfoSec812** [comment]
@TazSp Makes sense. I'd be happy to review a PR.

**Comment by InfoSec812** [comment]
This was resolved by #11
